app/presenters/finder_presenter.rb

```diff
@@ -125,7 +125,7 @@ def display_key_for_metadata_key(key)
   end
 
   def atom_url
-    ["#{slug}.atom", values.to_query].reject(&:blank?).join("?")
+    ["#{slug}.atom", values.to_query].reject(&:blank?).join("?") unless default_order
```

Comment:
I originally went that way, but the frontend part of it is checking if the `atom_feed` is nil or not, so I went down this route instead. It seemed like an extra step that wasn't needed but I guess it makes this clearer.